_episodes_rmd/01-introduction.Rmd

```diff
@@ -101,12 +101,12 @@ standalone software.
 
 ## Log on to RStudio Server
 
-Open a web browser and enter the IP address of your instance 
+Open a web browser and enter the DNS address you used to log in at the terminal 
```

Comment:
I'd just say URL here. It's true that it's a DNS hostname, but I don't think that is important for our purposes and might just add unnecessary cognitive load?

Reply:
Seeing as most folks are familiar with URL but maybe not with DNS...

Reply:
Great, thanks! I'll make this change and then merge.